modules/javafx.swing/src/main/java/com/sun/javafx/embed/swing/newimpl/SwingNodeInteropN.java

```diff
@@ -238,7 +238,8 @@ private static class SwingNodeDisposer implements DisposerRecord {
          SwingNodeDisposer(LightweightFrameWrapper ref) {
              this.lwFrame = ref;
          }
-         public void dispose() {
+         @Override
+        public void dispose() {
```

Comment:
in guthub ui these two lines appear misaligned.  are they really?

Reply:
Yeah, they are, the entire method is misaligned which is why the edit done by the quick fix is misaligned.  I'll fix the indent on the entire method.